policy-controller/grpc/src/lib.rs

```diff
@@ -761,26 +768,11 @@ fn convert_http_backend(backend: Backend) -> outbound::http_route::WeightedRoute
                 }),
                 filters: Default::default(),
             }),
-            weight: dst.weight,
         },
-        Backend::InvalidDst(dst) => outbound::http_route::WeightedRouteBackend {
+        outbound_route::Backend::InvalidDst(dst) => outbound::http_route::WeightedRouteBackend {
+            weight: dst.weight,
             backend: Some(outbound::http_route::RouteBackend {
-                backend: Some(outbound::Backend {
-                    metadata: None,
-                    queue: Some(default_queue_config()),
-                    kind: Some(outbound::backend::Kind::Balancer(
-                        outbound::backend::BalanceP2c {
-                            discovery: Some(outbound::backend::EndpointDiscovery {
-                                kind: Some(outbound::backend::endpoint_discovery::Kind::Dst(
-                                    outbound::backend::endpoint_discovery::DestinationGet {
-                                        path: dst.authority.clone(),
-                                    },
-                                )),
-                            }),
-                            load: Some(default_balancer_config()),
-                        },
-                    )),
-                }),
+                backend: None,
```

Comment:
I guess this is a functional change. Why would we need to send a backend on a failing route?

I can split this change out if desired.